app/services/invoices/payments/gocardless_service.rb

```diff
@@ -142,7 +147,14 @@ def invoice_payment_status(payment_status)
       def update_invoice_payment_status(payment_status)
         return unless Invoice::PAYMENT_STATUS.include?(payment_status&.to_sym)
 
-        invoice.update!(payment_status: payment_status)
+        invoice.update!(
+          payment_status: payment_status,
+          ready_for_payment_processing: payment_status.to_s == 'failed'
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)